[2024-05-21T14:35:32.397+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-21T14:35:32.422+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: statistical_calculations_dag.calculate_statistics scheduled__2021-10-25T00:00:00+00:00 [queued]>
[2024-05-21T14:35:32.431+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: statistical_calculations_dag.calculate_statistics scheduled__2021-10-25T00:00:00+00:00 [queued]>
[2024-05-21T14:35:32.432+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-21T14:35:32.448+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): calculate_statistics> on 2021-10-25 00:00:00+00:00
[2024-05-21T14:35:32.454+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=5736) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-21T14:35:32.455+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'statistical_calculations_dag', 'calculate_statistics', 'scheduled__2021-10-25T00:00:00+00:00', '--job-id', '1560', '--raw', '--subdir', 'DAGS_FOLDER/second_dag.py', '--cfg-path', '/tmp/tmpkh0qq27k']
[2024-05-21T14:35:32.456+0000] {standard_task_runner.py:63} INFO - Started process 5743 to run task
[2024-05-21T14:35:32.456+0000] {standard_task_runner.py:91} INFO - Job 1560: Subtask calculate_statistics
[2024-05-21T14:35:32.494+0000] {task_command.py:426} INFO - Running <TaskInstance: statistical_calculations_dag.calculate_statistics scheduled__2021-10-25T00:00:00+00:00 [running]> on host c420abfdc210
[2024-05-21T14:35:32.581+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='test' AIRFLOW_CTX_DAG_ID='statistical_calculations_dag' AIRFLOW_CTX_TASK_ID='calculate_statistics' AIRFLOW_CTX_EXECUTION_DATE='2021-10-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-10-25T00:00:00+00:00'
[2024-05-21T14:35:32.583+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-21T14:35:32.637+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-21T14:35:32.638+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-21T14:35:32.647+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=statistical_calculations_dag, task_id=calculate_statistics, run_id=scheduled__2021-10-25T00:00:00+00:00, execution_date=20211025T000000, start_date=20240521T143532, end_date=20240521T143532
[2024-05-21T14:35:32.670+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-21T14:35:32.692+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-21T14:35:32.696+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
